sph.py

```diff
@@ -1,6 +1,8 @@
 import numpy as np
 import matplotlib.pyplot as plt
 from scipy.special import gamma
+from numba import jit
```

Comment:
If ypu add a dependency, you should add it (and the version you have) in the requirements.txt